mdp/nodes/classifier_nodes.py

```diff
@@ -85,18 +104,39 @@ def _train(self, x, labels):
             self.offset_weight = new_offset
 
     def _label(self, x):
-        """Returns an array with class labels from the perceptron.
+        """  
+        :param x: A matrix having different variables on different columns
+            and observations on the rows.
```

Comment:
why not add ``:type x: numpy.ndarray``?

Reply:
IIRC I was not sure at the time whether MDP generalized to iterables. Due to limited knowledge of the specific modules, I was easy on said types. If that abstraction is not featured for this class, I will adapt the docstring. See http://mdp-toolkit.sourceforge.net/tutorial/iterables.html